qcelemental/molparse/to_string.py

```diff
@@ -143,6 +144,28 @@ def to_string(molrec: Dict,
             'coordinates': 'cartesian',
         }
 
+    elif dtype == 'molpro':
+
+        atom_format = '{elem}'
+        ghost_format = '{elem}'
+        umap = {'bohr': 'bohr', 'angstrom': 'angstrom'}
+
+        atoms = _atoms_formatter(molrec, geom, atom_format, ghost_format, width, prec, 2)
+
+        first_line = f"""{{{umap.get(units.lower())}}}"""
+        second_line = """geometry={"""
+        end_bracket = """}"""
+        smol = [first_line, second_line]
+        smol.extend(atoms)
+        smol.append(end_bracket)
+
+        if False in molrec['real']:
+            ghost_line = 'dummy,' + ','.join([str(idx + 1) for idx, real in enumerate(molrec['real']) if not real])
+            smol.append(ghost_line)
+
+        smol.append(f"set,charge={molrec['molecular_charge']}")
```

Comment:
if Molpro only accepts integer charges, will need to `int(...)` this.

Reply:
It seems like Molpro reads charge as a float, but I'm not sure if it still expects it to be an integer charge.

Reply:
if it reads as a float, then leave it. running or not is their responsibility then.